requirements/constraints.txt

```diff
@@ -24,8 +24,8 @@ django-cors-headers<3.0.0
 # It seems like django-countries > 5.5 may cause performance issues for us.
 django-countries==5.5
 
-# Removes deprecated get_ip function, which we still use (ARCHBOM-1329 for unpinning)
-django-ipware<3.0.0
+# 2.0.0 dropped support for Python 3.5
+django-pipeline<2.0.0
```

Comment:
Shouldn't this contstraint be removed all together?

Reply:
Removed it. This might have come in accidentally while resolving conflicts multiple times.